src/features/memoroom/memoRoomSaga.js

```diff
@@ -1,19 +1,27 @@
 import { all, call, put, takeEvery, fork } from "redux-saga/effects";
-import { getMemoRoomSuccess, getMemoRoomFailure } from "./memoRoomSlice";
+import {
+  getMemoListRequest,
+  getMemoListSuccess,
+  getMemoListFailure,
+} from "./memoRoomSlice";
+import memoApi from "../../utils/api/memo";
+
+function* getMemoList({ payload }) {
+  const { userId, memoroomId } = payload;
 
-function* getMemoRoom() {
   try {
-    const memoRoomData = yield call();
-    yield put(getMemoRoomSuccess(memoRoomData));
+    const memoRoomData = yield call(memoApi.getMemoList, userId, memoroomId);
```

Comment:
```suggestion
    const memoRoomData = yield call(memoApi.getMemoList, { userId, memoroomId });
```  
이렇게 한번에 묶어줘도 좋을 것 같아요!

Reply:
말씀주신 방법이 더 좋은것 같아요!! 바로 수정할게요!!